Documentation/Manual/LaTeX/chap-first-application.tex

```diff
@@ -294,7 +294,7 @@ \section{Adding Functionality}
 
 \begin{figure}
 \begin{center}
-\inputfig{ex2.pdf_t}
+\includegraphics{ex2-screenshot.png}
```

Comment:
Image is much smaller than expected. Try `\includegraphics[width=.6\linewidth]{ex2-screenshot.png}`.

Reply:
Fixed.